### ISSUE TITLE
Summoned simulacrum ignores player command

### ISSUE THREAD

**Issue by comiluv** [Gibberlings3/Enhanced-Powergaming-Scripts#27]
# Description
With "Enhanced Scripts for Simulacrum" installed, summoned simulacrums ignore player command and stops whatever they were doing every now and then

# Steps to reproduce the issue
1. Start a new game
2. Console in Vhailor's helm `C:CreateItem("wa2helm")`
3. Put it on a character
4. Summon a simulacrum with the helm
5. Command the simulacrum to move to another position
6. See error

# What happens
The simulacrum stops moving after a short amount of time

# What's expected to happen
The simulacrum finishes the player command and then resume its AI

# Screenshot
https://github.com/Gibberlings3/Enhanced-Powergaming-Scripts/assets/19642705/d096ea64-372f-4ff3-81d6-d311783ed0d7

# Additional info
Baldur's Gate 2 EE version 2.6.6.0
Enhanced Powergaming Scripts v12.5

# weidu.log
```
// Log of Currently Installed WeiDU Mods
// The top of the file is the 'oldest' mod
// ~TP2_File~ #language_number #component_number // [Subcomponent Name -> ] Component Name [ : Version]
~ENHANCED-POWERGAMING-SCRIPTS/ENHANCED-POWERGAMING-SCRIPTS.TP2~ #0 #15 // Install Enhanced Powergaming Scripts -> Accelerated Pre-Buffing Speed: v12.5
~ENHANCED-POWERGAMING-SCRIPTS/ENHANCED-POWERGAMING-SCRIPTS.TP2~ #0 #9 // Install Enhanced Scripts for Summoned Celestials: v12.5
~ENHANCED-POWERGAMING-SCRIPTS/ENHANCED-POWERGAMING-SCRIPTS.TP2~ #0 #10 // Install Enhanced Scripts for Simulacrum: v12.5
```

**Comment by morpheus562** [comment]
Thank you and I was able to figure out what was causing the issue. The script assigned to the simulacrum was working as desired, but there was an issue with the script assigning it to the simulacrum. I have been able to get it resolved, and the fix will be included in the next update.

**Comment by morpheus562** [comment]
Fixed in release v12.6.
